pegasus/sites.v3/code.org/public/volunteer/engineer/edit/splat.haml

```diff
@@ -31,18 +31,28 @@ theme: responsive
 
   .form-group
     .radio-row
-      %label.radio-inline
-        %input#unsubscribe-volunteer-annual{name: "unsubscribed_s", type: "radio", required: true, value: "untilhoc"}/
-        Stop sending me teacher requests until the next Hour of Code
-      %label.radio-inline
-        %input#unsubscribe-volunteer-annual{name: "unsubscribed_s", type: "radio", required: true, value: "forever"}/
-        Stop sending me teacher requests forever
-      %label.radio-inline
-        %input#unsubscribe-volunteer-annual{name: "unsubscribed_s", type: "radio", required: true, value: "resubscribe"}/
-        Send me teacher requests again
+      %div
+        %label.radio-inline
+          %input#unsubscribe-volunteer-annual{name: "unsubscribed_s", type: "radio", required: true, value: "untilhoc"}/
+          Stop sending me teacher requests until the next Hour of Code
+      %div
+        %label.radio-inline
+          %input#unsubscribe-volunteer-annual{name: "unsubscribed_s", type: "radio", required: true, value: "forever"}/
+          Stop sending me teacher requests forever
+      %div
+        %label.radio-inline
+          %input#unsubscribe-volunteer-annual{name: "unsubscribed_s", type: "radio", required: true, value: "resubscribe"}/
+          Send me teacher requests again
```

Comment:
1. I've already confirmed that I'm 18+
2. In the first form, I don't select anything for my subscription preferences
3. I click "Confirm" and it submits the form

Expected: It's a littttle funny that you can submit this form without selecting an option. I would expect to require selecting one of the radio buttons, even if it's the same subscription setting as they currently have. Although not blocking merging in this PR, just something that might be nice to clean up if it's a small change.